products/models.py

```diff
@@ -55,8 +68,33 @@ class Product(AuditTimeStamp):
     description = models.CharField(
         _("Product Description"), max_length=1000, blank=True
     )
+
+    image = models.ImageField(upload_to=image_path, default="default_image.png")
     name = models.CharField(_("Product Name"), max_length=100)
 
+    def get_price(self):
+        """
+        Returns price with currency
+        Returns:
+            (str): Value containing price and currency
+        """
+        return f"Rs. {self.price}"
+
+    def save(self, *args, **kwargs):
+        # pylint: disable=no-member
+        if self.id is not None:
+            product = Product.objects.get(id=self.id)
```

Comment:
How about we do `product = Product.objects.get(id=self.id)` and then check as it's still possible that we might not get a product but we run `product.image.name` in the next line which will throw error.

Reply:
I don't understand this comment Arslan bhai. If the id is not none that means a product does exist. If the product does not exist then `.get` will raise a `DoesNotExist` error. Please let me know if that is not the case

Reply:
I meant `filter` instead of `get`. e.g. `product = Product.objects.filter(id=self.id).first()` so in this case if id exists we'll get the product otherwise `None`.

Reply:
Done